File: 백준/Bronze/2562. 최댓값/최댓값.c
#include <stdio.h>

int main() {
    int array[9] = {0};
    int max = 0, p = 0;
	
    for (int i = 0; i < 9; i++) {
        scanf("%d", &array[i]);
        if (max < array[i]) {
            max = array[i];
            p = i + 1;
        }
    }
	
    printf("%d\n%d", max, p);
    return 0;
}

File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int[] array = new int[9];
        int max = 0, p = 0;

        Scanner scanner = new Scanner(System.in);
        
        for (int i = 0; i < 9; i++) {
            array[i] = scanner.nextInt();
            if (max < array[i]) {
                max = array[i];
                p = i + 1;
            }
        }
		
        System.out.println(max);
        System.out.println(p);
        scanner.close();
    }
}